public/app/plugins/datasource/prometheus/components/PromExploreQueryEditor.test.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
-import { shallow, render } from 'enzyme';
+import { shallow, mount } from 'enzyme';
 import PromExploreQueryEditor from './PromExploreQueryEditor';
```

Comment:
So do we want to use memoized version here? I thought you mentioned the memoization creates issues in the tests.

Reply:
Yes, changed it to non-memoized. Fix coming up in seconds.